.codecov.yml

```diff
@@ -2,49 +2,51 @@
 # After modifying this file, it might be worth to validate it with:
 # `curl --data-binary @.codecov.yml https://codecov.io/validate`
 
-# which branch to monitor and whether wait for ci success
-codecov:
-  branch: master
-  notify:
-    require_ci_to_pass: yes
+# NOTE: Codecov has been disabled for rose - too much effort to extract coverage from functional tests
 
-# define the colour bar limits here
-coverage:
-  precision: 2
-  round: down
-  range: "75...100"
+# # which branch to monitor and whether wait for ci success
```

Comment:
Why comment it out, rather than just removing, since the old stuff will be stored in the VC system.